### ISSUE TITLE
ocrd-cis-align producing unexpected xml

### ISSUE THREAD

**Issue by b2m** [cisocrgroup/ocrd_cis#76]
When using _ocrd-cis-align_ together with _ocrd-dinglehopper_ I noticed some unexpected behavior regarding the generated XML by _ocrd-cis-align_.

See qurator-spk/dinglehopper#37 for the way that led me here.

Here is the minimal workflow to reproduce the problem (also in the attached zip file as `workflow.sh`).
```bash
ocrd workspace init
ocrd workspace set-id "OCR-D-CIS-ALIGN-BUG"

ocrd workspace add --file-grp OCR-D-IMG --file-id OCR-D-IMG_f0001 --mimetype image/jpg --page-id PAGE_0001 OCR-D-IMG/FILE_0001.jpg

ocrd-olena-binarize -I OCR-D-IMG -O OCR-D-BIN -P impl sauvola -P k 0.2
ocrd-cis-ocropy-segment -I OCR-D-BIN  -O OCR-D-SEG-REG -P level-of-operation page
ocrd-tesserocr-recognize -I OCR-D-SEG-REG -O OCR-D-OCR-TESS1 -P textequiv_level word
ocrd-tesserocr-recognize -I OCR-D-SEG-REG -O OCR-D-OCR-TESS2 -P textequiv_level word
ocrd-cis-align -I OCR-D-OCR-TESS1,OCR-D-OCR-TESS2 -O OCR-D-ALIGN
```

I added a minimal example on how to reproduce the problem using docker and the [attached data](https://github.com/cisocrgroup/ocrd_cis/files/5452748/ocrd_cis_bug.zip):

```bash
 docker run --rm -it -v ${WORKSPACE}:/data -w /data -- ocrd/all:maximum bash workflow.sh
```

The unexpected part is, that the information from the text line from OCR-D-OCR-TESS2 is split into two XML nodes:
```xml
<pc:TextEquiv index="2" dataTypeDetails="OCR-D-OCR-TESS2/OCR-D-BIN_f0001_region0001_line0000"/>
<pc:TextEquiv conf="0.639380130767822" dataType="ocrd-cis-line-alignment" dataTypeDetails="OCR-D-OCR-TESS2/OCR-D-BIN_f0001_region0001_line0000">
  <pc:Unicode>上оrеm টрsum</pc:Unicode>
</pc:TextEquiv>
```

**Comment by cis-sp2021** [comment]
Thanks for reporting.  I'll look into that.

**Comment by finkf** [comment]
Thanks for reporting.  I'll look into this issue.

**Comment by finkf** [comment]
I could fix the issue see #78

**Comment by b2m** [comment]
Confirmed, I could not reproduce this issue with the newest version of OCR-D.
